src/compiler/program.ts

```diff
@@ -708,6 +711,20 @@ namespace ts {
                             (imports || (imports = [])).push(<LiteralExpression>moduleNameExpr);
                         }
                         break;
+                    case SyntaxKind.CallExpression:
+                        if (isJavaScriptFile && isRequireCall(node)) {
+
+                            let jsImports = (<CallExpression>node).arguments;
+                            if (jsImports) {
+                                imports = (imports || []);
+                                for (var i = 0; i < jsImports.length; i++) {

```

Comment:
`jsImports.length` is always one here or otherwise `isRequireCall` would have returned false. So why the loop?